src/resources/2015/MayoClinic.ts

```diff
@@ -0,0 +1,23 @@
+import { buy, getWorkshed, retrieveItem, toInt, use } from "kolmafia";
+import { $item, get, have } from "../..";
+import { withChoice } from "../../property";
+
+export const Mayo = {
+  nex: $item`Mayonex`,
+  diol: $item`Mayodiol`,
+  zapine: $item`Mayozapine`,
+  flex: $item`Mayoflex`,
+};
+
+export function mindMayo(mayo: Item, quantity: number): void {
```

Comment:
Can you rename this function to something less flavourful? Also add documentation